### PR TITLE
Fee Summary - Support Multiples and Mhr Results UX Improvements

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#12010

*Description of changes:*
* Updated Fee Summary to support additional Fee Charges
* Includes sub text for multiple charges and combined totals when additional fees are applied
* Applies Combined searches based off Lien selections
* Misc local data refactors to provide the FeeSummary reactivity to the props

* Updated Results/Review selections to preserve selections when navigating back for further edits

* Updated Unit testing

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/composables/fees/enums/feeSummaryDefaults.ts [hunk 1]
```diff
@@ -5,5 +5,7 @@ export enum FeeSummaryDefaults {
   DEFAULT_10 = 'feeDefault10',
   DEFAULT_500 = 'feeDefault500',
   SELECT_5 = 'feeSelect5',
-  SEARCH_5 = 'feeSearch5'
+  SEARCH_5 = 'feeSearch5',
+  SEARCH_8 = 'feeSearch8',
+  SEARCH_12 = 'feeSearch12'
```
Comment (cameron-eyds) [inline ppr-ui/src/composables/fees/enums/feeSummaryDefaults.ts:10]: @doug-lovett Not entirely sure how/if these line up with the api and what is expected there.

File: ppr-ui/src/composables/fees/enums/feeSummaryTypes.ts [hunk 2]
```diff
@@ -3,5 +3,6 @@ export enum FeeSummaryTypes {
   DISCHARGE = 'discharge',
   NEW = 'new',
   RENEW = 'renew',
-  MHSEARCH = 'manufactured_home_search'
+  MHSEARCH = 'manufactured_home_search',
+  MHR_COMBINED_SEARCH = 'combined_home_search'
```
Comment (cameron-eyds) [inline ppr-ui/src/composables/fees/enums/feeSummaryTypes.ts:7]: @doug-lovett Same comment as above, is the api aware of these types or potentially expect something different with regards to the combined search?

File: ppr-ui/src/composables/fees/enums/feeSummaryDefaults.ts (hunk 1 above)
Comment (doug-lovett) [reply ppr-ui/src/composables/fees/enums/feeSummaryDefaults.ts:10]: @cameron-freshworks No search fee info is exchanged between the UI and the API. The mhr api figures out the payment fling type codes and quantities from the token roles and the search selected matches.

File: ppr-ui/src/composables/fees/enums/feeSummaryTypes.ts (hunk 2 above)
Comment (doug-lovett) [reply ppr-ui/src/composables/fees/enums/feeSummaryTypes.ts:7]: @cameron-freshworks same answer as above.

File: ppr-ui/src/composables/fees/enums/feeSummaryDefaults.ts (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/composables/fees/enums/feeSummaryDefaults.ts:10]: Excellent!

File: ppr-ui/src/composables/fees/enums/feeSummaryTypes.ts (hunk 2 above)
Comment (cameron-eyds) [reply ppr-ui/src/composables/fees/enums/feeSummaryTypes.ts:7]: Thank you!

**Review by doug-lovett — APPROVED** [review]
Looks great.

**Review by dimak1 — APPROVED** [review]

**Pull request merged**